counsel.el

```diff
@@ -124,7 +124,9 @@ To skip the `executable-find' check, start the string with a space."
                      (car (split-string cmd)))))
       (or (and (stringp program)
                (not (string= program ""))
-               (executable-find program))
+               (if (< 27 emacs-major-version)
+                   (executable-find program (tramp-tramp-file-p default-directory))
```

Comment:
Note that `tramp-tramp-file-p` is not predefined, so it has to be loaded from `tramp.el`.

Would it be possible to use `file-remote-p` instead? Why did you pick `tramp-tramp-file-p` specifically?

Reply:
`file-remote-p` works too: https://github.com/abo-abo/swiper/pull/2618/commits/71bbc6c2dfd8f726f72df0cda9687476c5ce5127